addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -43,14 +43,24 @@ def test_mailing_statistics_sms(self):
         for record_idx in (0, 2, 3):
             self.gateway_sms_click(mailing, target_records[record_idx])
 
-        # check mailing statistics
         self.assertEqual(mailing.clicked, 3)
         self.assertEqual(mailing.delivered, 4)
-        self.assertEqual(mailing.received_ratio, 40)
         self.assertEqual(mailing.opened, 3)
-        self.assertEqual(mailing.opened_ratio, 30)
         self.assertEqual(mailing.sent, 10)
 
+        self.assertEqual(mailing.scheduled, 0)  # TODO: not zero values here
```

Comment:
can you make this non zero ? (with a commit before that one, to be sure it doesn't change with the fix)

Reply:
@std-odoo how many of these do you want to be non zero?
```
        self.assertEqual(mailing.scheduled, 0)  # TODO: not zero values here
        self.assertEqual(mailing.canceled, 0)
        self.assertEqual(mailing.process, 0)
        self.assertEqual(mailing.pending, 6)
        self.assertEqual(mailing.replied, 0)
        self.assertEqual(mailing.bounced, 0)
        self.assertEqual(mailing.failed, 0)
        ```
       all of them or just `mailing.scheduled` ?

Reply:
all

Reply:
@std-odoo i don't think we can do all. when testing sms we don't have reply flow created, we don't even detect it. same goes to many other flows: failed, process (those flows are for mailings). i will think how i can do scheduled, canceled and bounced. those

Reply:
can't you manually create the traces ?

Reply:
yes but it won't make sense, we are testing mailing feature on sms @std-odoo

Reply:
ah yeah indeed, we should do an other test so, sorry (so not in mass mailing sms, but just in mass mailing)

Reply:
@std-odoo done, i removed replied because it is already being tested in mail test. I only left `scheduled` and `process` because function doesn't allow me to capture those states, send function just does it right away and writes `sent` state at  the end

Reply:
@datonefaridze Can you do it in a different commit ? Before other commits (that way we can run the test before and after your changes)